DFC.ServiceTaxonomy.ApiFunction/CypherQueries/GetOccupationsWithMatchingSkillsDetailed.json

```diff
@@ -0,0 +1,13 @@
esco__skillReuseLevel]-(srl:skos__Concept) WHERE sx.uri in $skillList WITH soc, o, s, st, sm, rrl, srl, r, collect({uri:sx.uri, skill:head(sx.skos__prefLabel), alternativeLabels:coalesce(sx.skos__altLabel,[]), type:case head(st.skos__prefLabel) when 'skill' then 'competency' when 'knowledge' then 'knowledge' end, skillReusability:case head(srl.skos__prefLabel) when 'cross-sector skills and competences' then 'cross-sectoral' when 'sector specific skills and competences' then 'sector-specific' when 'occupation specific skills and competences' then 'occupation-specific' when 'transversal skills and competences' then 'transversal' end, relationshipType:'optional', lastModified:head(sx.dct__modified)}) as MatchingOptionalSkills, MatchingEssentialSkills MATCH (j:ncs__JobProfile) -- (o)<-[:esco__isEssentialSkillFor]-(sa) WHERE size(MatchingOptionalSkills) + size(MatchingEssentialSkills) >= $minimumMatchingSkills WITH o, collect({uri:sa.uri, skill:head(sa.skos__prefLabel), alternativeLabels:coalesce(sa.skos__altLabel,[]), type:case head(st.skos__prefLabel) when 'skill' then 'competency' when 'knowledge' then 'knowledge' end, skillReusability:case head(srl.skos__prefLabel) when 'cross-sector skills and competences' then 'cross-sectoral' when 'sector specific skills and competences' then 'sector-specific' when 'occupation specific skills and competences' then 'occupation-specific' when 'transversal skills and competences' then 'transversal' end, relationshipType:'essential', lastModified:head(sa.dct__modified)}) as AllEssentialSkills, s, st, sm, rrl, srl, soc, r, MatchingOptionalSkills,MatchingEssentialSkills, j.skos__prefLabel as JobProfile, j.uri as JobProfileUri, j.ncs__Description as JobProfileDescription OPTIONAL MATCH (j:ncs__JobProfile) -- (o)<-[:esco__isOptionalSkillFor]-(sa) WITH o, collect({uri:sa.uri, skill:head(sa.skos__prefLabel), alternativeLabels:coalesce(sa.skos__altLabel,[]), type:case head(st.skos__prefLabel) when 'skill' then 'competency' when 'knowledge' then 'knowledge' end, skillReusability:case head(srl.skos__prefLabel) when 'cross-sector skills and competences' then 'cross-sectoral' when 'sector specific skills and competences' then 'sector-specific' when 'occupation specific skills and competences' then 'occupation-specific' when 'transversal skills and competences' then 'transversal' end, relationshipType:'optional', lastModified:head(sa.dct__modified) }) as AllOptionalSkills, AllEssentialSkills, soc, MatchingOptionalSkills,MatchingEssentialSkills, JobProfile, JobProfileUri, JobProfileDescription RETURN { matchingOccupations: collect({uri:o.uri, occupation:o.skos__prefLabel, jobProfileTitle:JobProfile, jobProfileUri:JobProfileUri, jobProfileDescription:JobProfileDescription, matchingEssentialSkills:MatchingEssentialSkills, matchingOptionalSkills:MatchingOptionalSkills, socCode:soc.skos__prefLabel, occupationEssentialSkills:AllEssentialSkills, occupationOptionalSkills:AllOptionalSkills, lastModified:o.dct__modified}) } as results",
```

Comment:
No need to `head` the `skos__prefLabel` anymore, now that the whitelisting of array properties has made it into master.